vcd/resource_vcd_vapp_vm.go

```diff
@@ -257,6 +257,13 @@ func vmSchemaFunc(vmType typeOfVm) map[string]*schema.Schema {
 			Computed:    true,
 			Description: "Operating System type. Possible values can be found in documentation.",
 		},
+		"firmware": {
+			Type:         schema.TypeString,
+			Optional:     true,
+			Computed:     true,
+			ValidateFunc: validation.StringInSlice([]string{"efi", "bios"}, false),
+			Description:  "Firmware of the VM. Can be either EFI or BIOS, availability depending on the os_type argument. Changing the value when `power_on` is set to true, will cause a reboot of the VM.",
```

Comment:
Should we state that `bios` is default

Reply:
I updated the description, but didn't add the `Default:` parameter, as 37.0 and older versions don't have Firmware